hoss/dimension_utilities.py

```diff
@@ -181,7 +191,7 @@ def update_dimension_variables(
     # create the xy dim scales
     x_dim = np.arange(x_min, x_max, x_resolution)
 
-    # ascending versus descending..should be based on the coordinate grid
+    # The origin is the top left. Y values are in decreasing order.
```

Comment:
This is always the case? no exceptions?

Reply:
it is the case for SMAP. Let me see if I can make it more generalized.

Reply:
updated in this commit to check for latitude ascending/descending
https://github.com/nasa/harmony-opendap-subsetter/pull/15/commits/9c9f1908c770b15ff6535e43723230a892e7f7d1